website-1.x/pages/en/versions.js

```diff
@@ -26,8 +26,23 @@ function Versions(props) {
           <header className="postHeader">
             <h1>{siteConfig.title} Versions</h1>
           </header>
+          <h3 id="v2">Docusaurus v2</h3>
+          <p>
+            We recommend to use{' '}
+            <a href={`https://v2.docusaurus.io/`}>Docusaurus v2</a>.
+          </p>
+          <table className="versions">
+            <tbody>
+              <tr>
+                <th>2.x</th>
+                <td>
+                  <a href={`https://v2.docusaurus.io/`}>Documentation</a>
```

Comment:
Nit - redundant backticks and {}
```suggestion
                  <a href="https://v2.docusaurus.io">Documentation</a>
```

Reply:
thanks :)